README.md

```diff
@@ -25,12 +25,10 @@ Create a new `Settings` object to read settings from the environment variables:
 
 import { Settings, values } from 'node-env-settings';
 
-Settings.prefix = 'REACT_APP'
-
 export default new Settings({
   DEBUG: values.BooleanValue(false),
   MY_SETTING: values.Value('default value'),
-});
+}, 'REACT_APP');
```

Comment:
So you want to keep the order this way?

Reply:
I think so. I know it's not ideal but it's an optional param the first one is less so.

Reply:
Ah, I didn't realize it was optional. :+1: